src/Thenewboston/Common/Http/SimpleHttpRequestSender.cs

```diff
@@ -4,49 +4,76 @@
 
 namespace Thenewboston.Common.Http
 {
-    public class SimpleHttpRequestSender : IHttpRequestSender
+    public sealed class SimpleHttpRequestSender : IHttpRequestSender
     {
-        private HttpClient _client;
```

Comment:
Disposing HttpClient objects is a pretty bad idea, because of socket exhaustion. That's why it's good practice to keep it as a field and initialise it in the constructor.

Reply:
Ah yes that's true. In that case maybe this PR should be closed along with the issue. It may be beet to just add some information into the docs about handling timeouts and leave it at that.

Reply:
Alright then :D